### PR TITLE
Add basic gradle support

### PR DESCRIPTION
**Pull request by kotarak**
Second PR due to messed up sqashing. Sorry for the noise. :-/

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,7 @@
 
 ### New features
 
+* [#1373](https://github.com/clojure-emacs/cider/issues/1373): Add gradle support for `cider-jack-in`
```
Comment (bbatsov) [inline CHANGELOG.md]: This sentence is missing a `.`.

**Comment by bbatsov** [comment]
Seems you'll need to rebase your branch on top of `master`, as there's a merge conflict right now.

**Comment by kotarak** [comment]
Woohoo. Got the squash working.

**Comment by bbatsov** [comment]
Happy to hear this! :-)

**Comment by kotarak** [comment]
It made me like hg even more, though. I'm a patch man.

**Comment by bbatsov** [comment]
`git` is an acquired taste - had some issue with it originally, but at some point I grokked it and there's no going back to something else for me. :-)

**Comment by Malabarba** [comment]
I just never learned anything else, so git is the best VCS I've ever known! :smile:

**Pull request merged**